app/models/proposal.rb

```diff
@@ -80,7 +97,11 @@ def users
   # returns the Approval
   def add_approver(email)
     user = User.for_email(email)
-    self.approvals.create!(user_id: user.id)
+    if self.parallel? || self.approvals.empty?
+      self.approvals.create!(user_id: user.id, status: 'actionable')
+    else
+      self.approvals.create!(user_id: user.id)
+    end
```

Comment:
I mean that the `restart` method below is basically doing the same thing as these new lines, so we should move it out to a `set_actionable_approvals` method or something.